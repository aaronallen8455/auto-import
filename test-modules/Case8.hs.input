module Case8 where

main :: IO ()
main = do
  TIO.putStrLn "hi"
  TIO.putStrLn "bye"
