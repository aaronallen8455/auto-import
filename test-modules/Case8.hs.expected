module Case8 where

import qualified Data.Text.IO as TIO

main :: IO ()
main = do
  TIO.putStrLn "hi"
  TIO.putStrLn "bye"
